parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -110,6 +131,11 @@ public void writeTo(OutputStream out) throws IOException {
     }
 
     @Override
+    public byte[] getBytesUnsafe() {
+      return getBytes();
```

Comment:
Another option is to return the backing bytes and expose the start and length values.